### ISSUE TITLE
mysql start fails on ubuntu 14.04

### ISSUE THREAD

**Issue by snehitgajjar** [chef-cookbooks/mysql#455]
### Cookbook version
8.0
### Chef-client version
Chef: 12.13.37

### Platform Details
Ubuntu 14.04

### Scenario:
Trying to create and start default mysql-client and mysql-server

### Steps to Reproduce:
```
mysql_client 'default' do
  action :create
  package_version ''
end

mysql_service 'default' do
  port '3306'
  package_version ''
  initial_root_password rundeck_rdbms['rdbms']['dbpassword']
  action [:create, :restart]
end
```

### Expected Result:
Creation and installation of mysql-server and mysql-client

### Actual Result:

```
Running handlers:
       [2016-08-17T20:25:25+00:00] ERROR: Running exception handlers
       Running handlers complete
       [2016-08-17T20:25:25+00:00] ERROR: Exception handlers complete
       Chef Client failed. 79 resources updated in 03 minutes 52 seconds
       [2016-08-17T20:25:25+00:00] FATAL: Stacktrace dumped to /tmp/kitchen/cache/chef-stacktrace.out
       [2016-08-17T20:25:25+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
       [2016-08-17T20:25:25+00:00] ERROR: mysql_service[default] (rundeck::server_install line 243) had an error: Mixlib::ShellOut::ShellCommandFailed: mysql_service_manager_upstart[default] (/tmp/kitchen/cache/cookbooks/mysql/libraries/mysql_service.rb line 50) had an error: Mixlib::ShellOut::ShellCommandFailed: service[mysql-default] (/tmp/kitchen/cache/cookbooks/mysql/libraries/mysql_service_manager_upstart.rb line 68) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
       ---- Begin output of /sbin/start mysql-default ----
       STDOUT:
       STDERR: start: Unknown job: mysql-default
       ---- End output of /sbin/start mysql-default ----
       Ran /sbin/start mysql-default returned 1
       [2016-08-17T20:25:26+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
```

**Comment by snehitgajjar** [comment]
@jtimberman : Can you please take a look at this issue. I have tried lot to debug it. Only thing I found was missing attribute mysql_name [here](https://github.com/chef-cookbooks/mysql/blob/master/libraries/mysql_service_base.rb#L188). Thanks.

**Comment by someara** [comment]
Fixed up in 8.0.2
